enterprise/models.py

```diff
@@ -126,6 +127,7 @@ class Meta:
         help_text=_("Course catalog for the Enterprise Customer.")
     )
     active = models.BooleanField(default=True)
+    country = CountryField(null=True)
```

Comment:
We missed some of the AC's:

1. Required field
2. Default for current customers to US

Reply:
@zubair-arbi The field is currently 'required' and CS will update the current customers to US

> We missed some of the AC's:
> 
> 1. Required field
> 2. Default for current customers to US